### DISCUSSION TITLE
【求助】

### DISCUSSION THREAD

**Discussion by opqphp** [share121/inter-knot#4935] [其他]
为什么发帖或写评论会强制打开GitHub啊  还有就是米游社的表情包也不能用了  我记得一个月前还好好的  一个月没用什么都不会了
![Uploading illust_116799150_20240805_030623.png…]()

**Comment by hk0852** [comment]
米游社表情包本来是一位绳网用户自个搞得，因为现在绳网又再次更新了，要等那位创作者他自己来弄

**Reply by opqphp** [reply]
<img src="https://bbs-static.miyoushe.com/static/2023/05/23/a2fa805a0a782cdb8ab44ddaf8d5343d_4941628593528507734.png" alt="阿姬倒地了03" width="75" >绳网APP强制跳转GitHub怎么解决啊

**Reply by adventure-of-flame** [reply]
项目现阶段是基于Github来管理数据库的，应该是还没开发出帖子下方直接回复的功能

**Comment by HLXS-HLXS** [comment]
现在的绳网本质上是在GitHub里面开了个论坛，又搞了个爬虫把这个论坛里的帖子爬过去，你发帖当然要去论坛里发
